### ISSUE TITLE
That Which Creates Can Also Destroy

### ISSUE THREAD

**Issue by antonius123** [dalaranwow/dalaran-wow#5016]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
For this quest, http://www.wowhead.com/quest=12459/that-which-creates-can-also-destroy you are asked to throw a seed at 3 npcs and kill them. I was unable to complete this quest in Dragonblight, on Turgid the Vile and the Reanimated Frost Wyrm. First, Turgid the Vile was under the floor and untargetable by the seed. I made a ticket and Tyr brought him back on ground level so I can kill it. As for the Frost Wyrm, I throw the seed at it but it doesn't come at me to fight. It keeps patrolling in the sky at a range I can't reach as a melee class. So it is barely doable, unless I would ask some spellcaster to help me.

**Expected behaviour**: Tell us what should happen instead
Turgid the Vile should be in a small room in a ruined keep east of Wintergrade Keep. The Wyrm should come down from the sky to fight.

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and thank you for the report @antonius123,
Turgid the Vile should be fixed with #4917 (if approved) and Frost Wyrm with #1413
